modules/AOR_Reports/AOR_Report.php

```diff
@@ -715,7 +715,15 @@ function build_report_html($offset = -1, $links = true, $group_value = '', $tabl
                     if($att['function'] == 'COUNT' || !empty($att['params'])){
                         $html .= $row[$name];
                     } else {
-                        $html .= getModuleField($att['module'], $att['field'], $att['field'], 'DetailView',$row[$name],'',$currency_id);
+                        $field_string = getModuleField($att['module'], $att['field'], $att['field'], 'DetailView',$row[$name],'',$currency_id);
+                        if ($currency_id != '-99') {
+                            if (strpos($field_string, '$') !== false) {
+                                $currency = new Currency();
+                                $currency->retrieve($currency_id);
+                                $field_string = str_replace("$", $currency->symbol, $field_string);
```

Comment:
perhaps this fix should be in the Currency::retrieve() to apply every possible place where it also could be an issue.